code/modules/condor/cas_shuttle.dm

```diff
@@ -87,14 +87,14 @@
 	for(var/i in engines)
 		var/obj/structure/caspart/internalengine/engine = i
 		engine.cut_overlays()
-		var/image/engine_overlay = image('icons/turf/cas.dmi', engine.loc, "engine_idle", ABOVE_MOB_PROP_LAYER, pixel_x = engine.x_offset)
+		var/image/engine_overlay = image('icons/turf/cas.dmi', engine.loc, "engine_idle", ABOVE_MOB_LAYER, pixel_x = engine.x_offset)
 		engine.add_overlay(engine_overlay)
 
 ///Updates state and overlay to make te engines on
 /obj/docking_port/mobile/marine_dropship/casplane/proc/turn_on_engines()
 	for(var/i in engines)
 		var/obj/structure/caspart/internalengine/engine = i
-		var/image/engine_overlay = image('icons/turf/cas.dmi', engine.loc, "engine_idle", ABOVE_MOB_PROP_LAYER, pixel_x = engine.x_offset)
+		var/image/engine_overlay = image('icons/turf/cas.dmi', engine.loc, "engine_idle", ABOVE_MOB_LAYER, pixel_x = engine.x_offset)
```

Comment:
So I played a round today while this PR was test merged and debugged some stuff with an admin. The condor was broken. At first you can click on it and the cockpit would open and do the animation but when you click on it a 2nd time to enter, you would be denied. Instead you would get this message:

https://github.com/tgstation/TerraGov-Marine-Corps/blame/d855808c39343693be1647cc5f65b91d97783edc/code/modules/condor/cas_chair.dm#L73-L75

I suspect the overlays on the engine are causing it since it's coded via:

https://github.com/tgstation/TerraGov-Marine-Corps/blob/d855808c39343693be1647cc5f65b91d97783edc/code/modules/condor/cas_shuttle.dm#L115-L127

Which is stupid as hell. You should never use use an overlay to determine the state of an object.

Reply:
Correct, the emissive changes broke this